### PR TITLE
feat: run tests for wrappers when child directories have changed

### PR DESCRIPTION
**Pull request by viacheslav-rostovtsev**

### CODE REVIEW

File: .toys/ci.rb [hunk 1]
```diff
@@ -265,7 +265,14 @@ def find_changed_directories files
   dirs = Set.new
   files.each do |file|
     if file =~ %r{^([^/]+)/.+$}
-      dirs << Regexp.last_match[1]
+      dir = Regexp.last_match[1]
+      dirs << dir
+      if dir =~ %r{^(.+)-v[^-]+$}
```
Comment (dazuma) [inline .toys/ci.rb:270]: This is going to catch directories like `google-cloud-vision`

**Review by dazuma — CHANGES_REQUESTED** [review]

File: .toys/ci.rb [hunk 2]
```diff
@@ -288,7 +295,7 @@ def filter_gem_dirs dirs
     else
       false
     end
-  end.sort
+  end.uniq.sort
```
Comment (dazuma) [inline .toys/ci.rb:298]: The `uniq` shouldn't be necessary. The input `dirs` should already be uniquified because it is populated from a set.

**Review by dazuma — CHANGES_REQUESTED** [review]

File: .toys/ci.rb (hunk 1 above)
Comment (viacheslav-rostovtsev) [reply .toys/ci.rb]: :+1:

File: .toys/ci.rb (hunk 2 above)
Comment (viacheslav-rostovtsev) [reply .toys/ci.rb]: :+1:

**Review by dazuma — APPROVED** [review]

**Pull request merged**